### ISSUE TITLE
Avoid loading assets from Google

### ISSUE THREAD

**Issue by jancborchardt** [nextcloud/nextcloud.com#606]
It’s a bit antithetical that we load fonts and jQuery from Google in some places, whereas we can really easily serve them ourselves. Is there any specific reason for that?

If not, here’s a PR to load the fonts from our own site: https://github.com/nextcloud/nextcloud.com/pull/605

And I saw some places where jQuery and other libs are loaded from googleapis, would be good to fix that too.

**Comment by jospoortvliet** [comment]
Yeah, I'd love to clean this up, it is now a combination of self and not. The javascript is loaded from a CDN first, then locally, that's clever, but for the css and fonts this doesn't work. I just don't know how to fix that...
